src/components/simpleHeader/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FC } from 'react';
+
+import { chakra, Flex, useMultiStyleConfig } from '@chakra-ui/react';
+
+import { CloseIcon } from '../../index';
+
+interface Props {
+  title: string;
+  url: string;
+}
+
+const SimpleHeader: FC<Props> = ({ title, url }) => {
+  const styles = useMultiStyleConfig(`SimpleHeader`);
+
+  const Component = chakra('header', { baseStyle: styles.header });
+
+  return (
+    <Component>
+      <Flex
+        minH={{ xs: 'xl', lg: '2xl' }}
+        paddingX={{ xs: 'md', lg: '4xl' }}
+        paddingY="xl"
+        align="center"
+        justify="space-between"
+        border="1px"
+        borderX="none"
+        borderTop="none"
+        borderBottomColor="gray.200"
```

Comment:
Adding the separator to the component feels off to me as I would expect that to be part of the page layout

Reply:
The separator is part of the simple header as figma shows. I think, it makes more sense to have the separator in the simple header component because in the pages we must not add any and depending on the margins we set on the page, the space will be also applied on the separator.

In case that we are implementing the page and we find that it'd be easier adding that on the page itself instead in the component, we could always adapt it to our needs. Besides, it's hard to adapt a component to all the cases without seeing the big picture.

Reply:
That's what I had in mind when thinking of integrating the component to a page layout as by adding the button border and spacing to the header it will overflow the layout the page will need and we need to tick with the integration.


![Screenshot 2022-07-29 at 11 46 28](https://user-images.githubusercontent.com/3371760/181733424-e574bd29-33b4-47c9-807d-4a0d872fe4c3.png)


We touched on creating layouts when we discussed the components library: https://github.com/smg-automotive/au-docs/discussions/3 so what I think when looking at the figma would be a layout something like that

https://miro.com/app/board/uXjVOi4oo7Y=/
![image](https://user-images.githubusercontent.com/3371760/181735039-50a8ba09-3a59-4dad-a1a8-00bea4795167.png)